core/src/main/scala/kafka/admin/AdminUtils.scala

```diff
@@ -707,6 +708,7 @@ object AdminUtils extends Logging with AdminUtilities {
   private def getBrokerInfoFromCache(zkUtils: ZkUtils,
                                      cachedBrokerInfo: scala.collection.mutable.Map[Int, Broker],
                                      brokerIds: Seq[Int]): Seq[Broker] = {
+//    TODO: This buffer is ignored, can be deleted?
```

Comment:
Do you mean the `failedBrokerIds` variable? I think we should use it at the end (e.g. line 728) for printing a warnning log4j entry.